frontend/src/games/aim-trainer-game.tsx

```diff
@@ -60,23 +64,30 @@ function styleElement(
 ) {
   element.className = 'w-24 h-24 bg-sky-500 rounded-full';
   element.style.position = 'absolute';
-  console.log(`x: ${x}, y: ${y}`);
   element.style.top = `${y + BORDER / 2}px`;
   element.style.left = `${x + BORDER / 2}px`;
 }
 
 const AimTrainerGame = () => {
-  const [gameIsStarted, dotsAreSpawning] = useState(false);
+  const [gameIsStarted, setGameIsStarted] = useState(false);
   const [difficulty, setDifficulty] = useState(Difficulty.EASY);
   const [isLoading, setIsLoading] = useState(false);
   const [dotsLeft, setDotsLeft] = useState(0);
   const [score, setScore] = useState(0);
 
+  const dots: MutableRefObject<Array<HTMLDivElement>> = useRef([]);
+
   function removeElement(element: HTMLDivElement) {
     element.remove();
     setDotsLeft((prevDots) => prevDots - 1);
   }
 
+  function removeAllDots() {
+    dots.current.forEach(removeElement);
+
```

Comment:
Nzn ar cia reikia tuscios eilutes. Jei kazka pries mergindamas dar keisi, consider removing it, jei ne, tai whatever nbd.

Reply:
Okey, paremovinsiu